#14 Footer с социальными кнопками (Нижняя часть сайта)

Androooooooid opened the pull request:
Вроде как то так

nastyapottapova committed:
Add files via upload

nastyapottapova committed:
Merge pull request #16 from iatudev/nastya_pottapova

Шапка с меню и логотипом+Баннер

Andrei7321 committed:
Coffee

Androooooooid committed:
Footer с социальными кнопками (Нижняя часть сайта)

Вроде как то так

rsagiev commented on footer.html:
```diff
@@ -0,0 +1,25 @@
+<footer>
+	<div class="container">
+         <div class="social">
+            <a href="http://facebook.com/" class=".facebook-icon"><img src="image/facebook-icon.svg"></a>
```
Название класса без точки

rsagiev commented on footer.html:
```diff
@@ -0,0 +1,25 @@
+         <div class="social">
+            <a href="http://facebook.com/" class=".facebook-icon"><img src="image/facebook-icon.svg"></a>
+			<a href="http://twitter.com/" class="twitter-icon"><img src="image/twitter-icon.svg"></a>
+	        <a href="http://instagram.com/" class="Instagram-icon"><img src="image/instagram-icon.svg"></a>
```
Заглавную убери

rsagiev commented on footer.html:
```diff
@@ -0,0 +1,25 @@
+			<a href="http://twitter.com/" class="twitter-icon"><img src="image/twitter-icon.svg"></a>
+	        <a href="http://instagram.com/" class="Instagram-icon"><img src="image/instagram-icon.svg"></a>
+	     </div>
+	     <div class="Copyright">
```
Табы на пробелы

rsagiev commented on image/facebook-icon.svg:
```diff
@@ -0,0 +1,7 @@
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ width="14px" height="26px">
+<path fill-rule="evenodd"  fill="rgb(255, 255, 255)"
+ d="M13.607,0.180 C13.182,0.118 11.719,-0.007 10.020,-0.007 C6.465,-0.007 4.027,2.149 4.027,6.102 L4.027,9.508 L0.016,9.508 L0.016,14.132 L4.027,14.132 L4.027,25.990 L8.840,25.990 L8.840,14.132 L12.836,14.132 L13.449,9.508 L8.840,9.508 L8.840,6.555 C8.840,5.227 9.202,4.305 11.137,4.305 L13.607,4.305 L13.607,0.180 Z"/>
```
Если делаешь с переносами, переноси везде

Androooooooid replied:
Вобще не понял вопроса с переносом эти картинки формата svg скачал с zeplin

rsagiev replied:
@Androooooooid первый тег атрибуты с новых строк, второй в одну строку. надо к одному стилю привести

rsagiev commented on style.css:
```diff
@@ -1,84 +1,25 @@
-	margin:0;
-	padding:0;
+.container {
+	width: 1600;
```
респонзива не будет, а надо

rsagiev commented on style.css:
```diff
@@ -1,84 +1,25 @@
-	
+.Copyright{
+    text-align: center;
+	width: 437px;
```
Табы на пробелы везде заменить надо

Androooooooid replied:
Почему табы нельзя оставлять зачем именно пробелы есть человеческое объяснение? Мне так проще понимать код.

rsagiev replied:
@Androooooooid делается для того, чтобы код везде был одинаков. в любой нормальной компании есть стандарты разработки и мы на паре выбрали пробелы

Androooooooid committed:
Исправил точку

Androooooooid committed:
#14 Фиксировал респонзив 

теперь он будет:D

Androooooooid committed:
#14 Footer с социальными кнопками (Нижняя часть сайта)

fix class

Androooooooid committed:
#14 Footer с социальными кнопками (Нижняя часть сайта)

исправил табы на пробелы в ксс и подкорректировал респонсив теперь корректироваться под любой размер

Androooooooid committed:
#14 Footer с социальными кнопками (Нижняя часть сайта) 

табы на пробелы

rsagiev commented on footer.html:
```diff
@@ -0,0 +1,25 @@
+
+
+<footer>
+ <div class="container">
```
перенести блок в индекс файл надо

nastyapottapova committed:
Delete coffee-shop.jpg

nastyapottapova committed:
Delete Be-latte-logo.png

nastyapottapova committed:
Add files via upload

Androooooooid committed:
#14 Footer с социальными кнопками (Нижняя часть сайта)

изменил название footer.html на index.html

Androooooooid committed:
Delete footer.html
